src/ansi-c/ansi_c_scope.cpp

```diff
@@ -10,6 +10,11 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include "ansi_c_scope.h"
 
+std::ostream &operator<<(std::ostream &os, ansi_c_id_classt c)
+{
+  return os << static_cast<int>(c);
+}
+
```

Comment:
This does not have any comment header and, really, should print proper strings if taking the time to define the `<<` operator.